ansible/library/check_bgp_ipv6_routes_converged.py

```diff
@@ -18,6 +22,30 @@ def get_bgp_ipv6_routes(module):
     return json.loads(out)
 
 
+def perform_action(module, action, connection_type, targets, all_neighbors):
```

Comment:
the function name is very generic, so will be better to do 2 things:

1. Make it private to this module - adding "_" prefix.
2. Make it as explicit as possible, even something like: `_perform_device_action` or `_perform_startup_shutdown_action`.